sdk/tables/Azure.Data.Tables/src/TableGeoReplicationInfo.cs

```diff
@@ -8,7 +8,7 @@ namespace Azure.Data.Tables.Models
 {
     /// <summary> The GeoReplication. </summary>
     [CodeGenModel("GeoReplication")]
```

Comment:
Is this attribute still correct?  I looks like the generated model name has `Info` as a suffix as well.

Reply:
`CodeGenModel` is used to specify the swagger name of the model, so that [autorest ](https://github.com/Azure/autorest.csharp#rename-a-model-class)knows which one you intended to rename.